### ISSUE TITLE
Paging broken

### ISSUE THREAD

**Issue by makeworld-the-better-one** [sharkdp/bat#1639]
**Describe the bug you encountered:**

When using bat, it does not respond to mouse scrolling, and exiting bat does not clear the visible file buffer out of the terminal.

**What did you expect to happen instead?**

Bat is paging with `less`, and so I expect it to act exactly like `less`. When I use `less`, it responds to mouse scrolling, and the file is not visible after leaving `less` with <kbd>q</kbd>.

**How did you install `bat`?**

From the Arch Linux package repos. Specifically `community/bat`.

---

**bat version and environment**

#### Software version

bat 0.18.0

#### Operating system

Linux 5.11.16-arch1-1

#### Command-line

```bash
bat dither.go --diagnostic 
```

#### Environment variables

```bash
SHELL=/usr/bin/zsh
PAGER=less
BAT_PAGER=<not set>
BAT_CACHE_PATH=<not set>
BAT_CONFIG_PATH=<not set>
BAT_OPTS=<not set>
BAT_STYLE=<not set>
BAT_TABS=<not set>
BAT_THEME=<not set>
XDG_CONFIG_HOME=<not set>
XDG_CACHE_HOME=<not set>
COLORTERM=truecolor
NO_COLOR=<not set>
MANPAGER=<not set>
```

#### Config file

Could not read contents of '/home/makeworld/.config/bat/config': No such file or directory (os error 2).

#### Compile time information

- Profile: release
- Target triple: x86_64-unknown-linux-gnu
- Family: unix
- OS: linux
- Architecture: x86_64
- Pointer width: 64
- Endian: little
- CPU features: fxsr,sse,sse2
- Host: x86_64-unknown-linux-gnu

#### Less version

```
> less --version 
less 581.2 (PCRE2 regular expressions)
Copyright (C) 1984-2021  Mark Nudelman

less comes with NO WARRANTY, to the extent permitted by law.
For information about the terms of redistribution,
see the file named README in the less distribution.
Home page: https://greenwoodsoftware.com/less
```

**Comment by makeworld-the-better-one** [comment]
Just want to add that bat did not always do this. I'm not sure what caused the change, but I remember this working fine in the past.

**Comment by eth-p** [comment]
This is caused by `bat` failing to parse the version of `less` (fix coming in #1269).

Basically, `bat` extracts the version string `581.2` and tries to parse it as an integer. It encounters the `.` and fails, which causes `bat` to assume only the oldest version of `less` is available.

**Comment by makeworld-the-better-one** [comment]
Sounds good, thanks! Hope a new version will come soon.

**Comment by sharkdp** [comment]
closing this, as #1629 by @aswild has been merged.

**Comment by sharkdp** [comment]
Fixed in bat 0.18.1
